Makefile

```diff
@@ -45,10 +63,9 @@ manifest:
 	sed -i "s#image: ghcr.io/.*kured.*#image: ghcr.io/$(DH_ORG)/kured:$(VERSION)#g" kured-ds-signal.yaml
 	echo "Please generate combined manifest if necessary"
 
-test:
-	echo "Running go tests"
-	go test ./...
-	echo "Running golint on pkg"
-	golint ./pkg/...
-	echo "Running golint on cmd"
-	golint ./cmd/...
+test: bootstrap-tools
+	echo "Running short go tests"
+	go test -test.short -json ./... > test.json
+	echo "Running shellcheck"
+	find . -name '*.sh' -exec .tmp/shellcheck  {} \;
```

Comment:
Path needs to be adjusted.

Reply:
It doesn't seem to fail either: https://github.com/kubereboot/kured/actions/runs/11305350571/job/31444749412?pr=982#step:4:116

Reply:
I am not a Makefile expert, this behaviour was already present. Any idea?